Objects/memoryobject.c

```diff
@@ -1589,7 +1600,11 @@ memory_getbuf(PyObject *_self, Py_buffer *view, int flags)
 
 
     view->obj = Py_NewRef(self);
+    #ifdef Py_GIL_DISABLED
+    _Py_atomic_add_ssize(&self->exports, 1);
+    #else
     self->exports++;
+    #endif
```

Comment:
Formatting: the convention for us is to keep preprocessor macros unindented:
```suggestion
#ifdef Py_GIL_DISABLED
    _Py_atomic_add_ssize(&self->exports, 1);
#else
    self->exports++;
#endif
```

Reply:
Hi @ZeroIntensity ,

Got it! 😊

I have updated the macro indent here.

Best Regards,
Edward